File: pepper-farm/backend/Models/PepperVariety.cs
namespace PepperFarm.Api.Models;

public class PepperVariety
{
    public int PepperId { get; set; }
    public string PepperName { get; set; } = string.Empty;
    public string? ScientificName { get; set; }
    public int? HeatLevelScovilleMin { get; set; }
    public int? HeatLevelScovilleMax { get; set; }
    public decimal? OptimalSoilMoistureMin { get; set; }
    public decimal? OptimalSoilMoistureMax { get; set; }
    public decimal? OptimalTempMinC { get; set; }
    public decimal? OptimalTempMaxC { get; set; }
    public decimal? OptimalSunlightHours { get; set; }
    public string? ImageUrl { get; set; }
    public string? Zone { get; set; }
    public string? GeneralDescription { get; set; }
    public bool IsActive { get; set; } = true;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}

File: pepper-farm/backend/models/pepper_variety.py
from sqlalchemy import Column, Integer, String, Boolean, DateTime, Numeric
from sqlalchemy.sql import func
from database import Base

class PepperVariety(Base):
    __tablename__ = "PepperVarieties"

    PepperId               = Column(Integer, primary_key=True, autoincrement=True)
    PepperName             = Column(String(100), nullable=False, unique=True)
    ScientificName         = Column(String(150), nullable=True)
    HeatLevelScovilleMin   = Column(Integer, nullable=True)
    HeatLevelScovilleMax   = Column(Integer, nullable=True)
    OptimalSoilMoistureMin = Column(Numeric(5, 2), nullable=True)
    OptimalSoilMoistureMax = Column(Numeric(5, 2), nullable=True)
    OptimalTempMinC        = Column(Numeric(5, 2), nullable=True)
    OptimalTempMaxC        = Column(Numeric(5, 2), nullable=True)
    OptimalPARMin          = Column(Numeric(7, 2), nullable=True)
    OptimalPARMax          = Column(Numeric(7, 2), nullable=True)
    ImageUrl               = Column(String(500), nullable=True)
    Zone                   = Column(String(500), nullable=True)
    GeneralDescription     = Column(String(1000), nullable=True)
    IsActive               = Column(Boolean, nullable=False, default=True)
    CreatedAt              = Column(DateTime, nullable=False, server_default=func.sysutcdatetime())
